README.md

```diff
@@ -1,29 +1,31 @@
-[![Build Status](https://travis-ci.org/tvkitchen/tv-kitchen.svg?branch=master)](https://travis-ci.org/tvkitchen/tv-kitchen)
+[![Build Status](https://travis-ci.org/tvkitchen/tv-kitchen.svg?branch=master)](https://travis-ci.org/tvkitchen/countertop)
 
-# TV Kitchen
+# TV Kitchen Countertop
 
 The [TV Kitchen](https://tv.kitchen) is a playground that turns TV into data (transcripts, facial recognition, etc.) so that newsrooms can help their staff and readers better understand what is happening across various local and cable TV news outlets.
 
+The Countertop is the entry point for developers who want to set up a TV Kitchen.
+
 ## Setting up the project
 
-If you are interested in running the TV Kitchen as a developer please review our [install instructions](docs/INSTALL.md).
+If you are interested in running a TV please review our [install instructions](docs/INSTALL.md).
```

Comment:
This seems garbled. "running a TV" doesn't seem to be what you mean to say!

Reply:
![image](https://user-images.githubusercontent.com/208884/91448661-28526800-e848-11ea-92cf-563d75b2ed8f.png)